src/UNIV2LPOracle.sol

```diff
@@ -96,8 +96,13 @@ contract UNIV2LPOracle {
     }
 
     address public immutable src;   // Price source
-    uint16  public hop = 1 hours;   // Minimum time inbetween price updates
-    uint64  public zzz;             // Time of last price update
+
+    // hop and zph are packed into single slot to reduce SLOADs;
+    // this outweighs the cost from added bitmasking operations.
+    uint16  public stopped;         // Stop/start ability to update
+    uint16  public hop = 1 hours;   // Minimum time in between price updates
```

Comment:
I wonder if we should cut 8 bits off of zph and make hop a uint24. 

I think uint16 gives us a max of like 18 hours, which feels somewhat short.

Reply:
Certainly could do that. `hop` historically has been a `uint16` though so we've been okay with it to this point.
Actually, it probably makes more sense to make `stopped` a `uint8` than it does to take bits off of `zph`. I'll just do that, @NiklasKunkel can object if he wants to keep `hop` at 16 bits.

Reply:
Change pushed.

Reply:
Ended up keeping `hop` at 16 bits for backwards compatibility reasons, since we just very nearly depended on it for the `ClipperMom` permissionless circuit breaker. Would be very annoying if we wound up with mixed oracle APIs.

Reply:
(and there's no strong business reason to increase the width at the moment)